lib/checks/label/hidden-explicit-label.js

```diff
@@ -4,7 +4,9 @@ if (node.getAttribute('id')) {
 	const label = root.querySelector(`label[for="${id}"]`);
 
 	if (label && !axe.commons.dom.isVisible(label)) {
```

Comment:
We should change this so it does `!axe.commons.dom.isVisible(label, true))`.

Here's a test case for it:

```js
it('should fail when the label has aria-hidden=true', function() {
	var html = '';
	html += '<div>';
	html += '  <label for="target" aria-hidden="true">';
	html += '    Hello world';
	html += '  </label>';
	html += '  <input id="target">';
	html += '</div>';
	var args = checkSetup(html, {}, '#target');
	assert.isTrue(check.evaluate.apply(check, args));
});
```

Reply:
Good call! Updated.